api_tests/institutions/views/test_institution_user_metric_list.py

```diff
@@ -145,3 +162,10 @@ def test_get(self, app, url, user, user2, admin, institution, populate_counts):
     def test_filter(self, app, url, admin, populate_counts):
         resp = app.get(f'{url}?filter[department]=Psychology dept', auth=admin.auth)
         assert resp.json['data'][0]['attributes']['department'] == 'Psychology dept'
+
+    def test_sort_and_pagination(self, app, url, admin, populate_more_counts):
+        resp = app.get(f'{url}?sort=user_name&page[size]=1&page=2', auth=admin.auth)
+        assert resp.status_code == 200
+        assert resp.json['links']['meta']['total'] == 3
```

Comment:
I think you need at least 11 to properly test this bug, because it relies on the default return size of Elastic. If someone were to remove https://github.com/CenterForOpenScience/osf.io/pull/9393/files#diff-f2e5b2e36d4c730f6167cefa2d18b901R289 this test would still pass.